### DISCUSSION TITLE
Change Notice

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#515] [Q&A] [answered]
Have you defined which changes in the notice (which information) should generate a "change notice" or a new notice?

Thanks in advance

**Comment by ferrakaop** [comment, marked as answer]
Every Change notice is a new notice, i.e. a new XML with a Change section and a link to the notice it is changing.  You will need a Change notice whatever the information you want to change.

If you're asking about which fields can be changed with a Change notice, we've put the preliminary list in the **draft** minutes of the May workshop - search for "10 fields" at https://op.europa.eu/en/web/ted-eforms/minutes-5th-eforms - the rules are not implemented yet but you can also use this as an indication of what rules we intend to apply in future.

**Reply by rengit62** [reply]
Hi Karl first of all thanks for the link !
I've got a doubt. If , for example, BT-23 cannot be changed in a change notice, the same should be for BT-262, BT-531 .. and so on.
I mean the list of '10 fields' has to be 'interpreted'
I'm doing it wrong ?
Bye
Renato

**Reply by ferrakaop** [reply]
The list of fields is still an indicative draft but we don't have any plans to over-interpret the rules.  For example, you shouldn't change BT-23 Main Nature but we don't see any reason not to allow changes to BT-262 Main Classification Code or BT-531 Additional Nature if needed.

Note that the Change notice still needs to respect whatever rules apply to the notice, e.g. if you change CPV code, it would still need to fit into the CPV codes that are valid for the Main Nature.
